arlo-client/tsconfig.json

```diff
@@ -13,15 +13,20 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve",
     "types": [
       "node",
       "@wdio/sync",
       "@wdio/jasmine-framework",
       "@types/jest",
       "@testing-library/jest-dom"
     ],
-    "noUnusedLocals": true
+    "jsx": "preserve"
   },
-  "include": ["src"]
+  "include": ["src"],
+  "exclude": [
+    "**/_mocks.ts",
+    "**/*.test.*",
+    "**/testUtilities.ts",
+    "src/test/**"
+  ]
```

Comment:
@jonahkagan I just discovered that this breaks intellisense for VSCode in test files. It doesn't recognize `describe` et al until I remove these lines. Is there a way to keep them for development but not have them apply on heroku?

Reply:
https://devcenter.heroku.com/articles/slug-compiler#ignoring-files-with-slugignore

Reply:
Ugh the problem that never ends!